### PR TITLE
Design of List View layout  in Resource Page

### LINKED ISSUE #9090: Update the List view Layout in Resources
The current list view in the Resource layout needs to be updated to ensure a consistent design that aligns with the list layout used in the Shifting view. This update aims to enhance visual consistency and improve the user experience across different sections of the application
Before:
Image

Refernce:
Image

### PR DESCRIPTION
**Pull request by AnveshNalimela**
## Proposed Changes

- Fixes #9090 
-Changed the List View into List format which includes imp deatils of resource card
Before:
{C1EF9F7C-2E0E-497A-9300-75F16E17E354}
 After:
{62C98BB2-2BAA-467A-89C5-5E7A2E9E16B3}
{7E14FCCB-20BD-4911-B43D-3723445D0B45}

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

## Summary by CodeRabbit

- **New Features**
	- Added new labels for "Approved," "Pending," "Resource Status," and "Transportation" in English localization.
	- Enhanced the Resource List component with a new grid layout displaying detailed resource information, including title, category, status, and last modified date.

- **Bug Fixes**
	- Improved rendering logic for resource status with conditional styling.

- **Style**
	- Updated styling and layout for the Resource List to enhance user experience, including adjustments for better visual clarity and organization.

### CODE REVIEW

File: src/components/Resource/ResourceList.tsx [hunk 1]
```diff
@@ -123,10 +234,24 @@
...
-              {data?.results && showResourceCardList(data?.results)}
+            <div className="mx-5 mt-5 grid w-full gap-2 border-b-2 border-gray-100 p-4 text-sm font-medium sm:grid-cols-1 md:grid-cols-1 lg:grid-cols-5">
+              <div className="col-span-1 uppercase sm:text-center md:text-center lg:block lg:text-left">
+                {t("resource")}
+              </div>
+              <div className="col-span-1 hidden text-left uppercase sm:hidden md:hidden lg:block">
+                {t("LOG_UPDATE_FIELD_LABEL__patient_category")}
+              </div>
+              <div className="col-span-1 hidden text-left uppercase sm:hidden md:hidden lg:block">
+                {t("consent__status")}
```
[the remark this answers is not in the document]
Comment (AnveshNalimela) [reply src/components/Resource/ResourceList.tsx:258]: Adding another translation for same does not make any sense

File: src/components/Resource/ResourceList.tsx [hunk 2]
```diff
@@ -46,17 +46,130 @@
   });
 
   const showResourceCardList = (data: ResourceModel[]) => {
+    if (loading) {
+      return <Loading />;
+    }
```
[the remark this answers is not in the document]
Comment (AnveshNalimela) [reply src/components/Resource/ResourceList.tsx]: Fixed

File: src/components/Resource/ResourceList.tsx [hunk 3]
```diff
@@ -46,17 +46,130 @@ export default function ListView() {
...
+            <div className="address mt-1">
+              <dd className="text-xs font-medium leading-5">
+                {resource.sub_category || "--"}
+              </dd>
+            </div>
+          </div>
+
+          <div className="col-span-1 flex flex-col px-3 text-left">
+            <div className="3xl:flex-row mb-2 flex gap-2 sm:flex-row md:flex-row lg:flex-col xl:flex-row 2xl:flex-row ">
+              {resource.status == "TRANSPORTATION TO BE ARRANGED" ? (
```
[the remark this answers is not in the document]
Comment (AnveshNalimela) [reply src/components/Resource/ResourceList.tsx]: Done

File: src/components/Resource/ResourceList.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (AnveshNalimela) [reply src/components/Resource/ResourceList.tsx:258]: No need to add new translation

**Comment by rithviknishad** [comment]
add readable titles for the PR @AnveshNalimela

**Comment by AnveshNalimela** [comment]
@rithviknishad  Could you check on these PR a

**Review by rithviknishad — APPROVED** [review]

File: src/components/Resource/ResourceList.tsx [hunk 4]
```diff
@@ -48,15 +48,125 @@ export default function ListView() {
...
+
+          <div className="col-span-1 flex flex-col px-3 text-left">
+            <div className="3xl:flex-row mb-2 flex gap-2 sm:flex-row md:flex-row lg:flex-col xl:flex-row 2xl:flex-row ">
+              {resource.status === "TRANSPORTATION TO BE ARRANGED" ? (
+                <dt
+                  title={t("resource_status")}
+                  className="w-3/4 mt-1 h-fit flex h-5 shrink-0 items-center rounded-full px-2 py-0.5 text-xs font-medium leading-4  overflow-hidden whitespace-nowrap text-ellipsis truncate bg-gray-200"
+                >
+                  <CareIcon icon="l-truck" className="mr-2" />
+                  <dd className="text-sky-600 truncate">{resource.status}</dd>
```
Comment (shivankacker) [inline src/components/Resource/ResourceList.tsx]:
```suggestion
                  <dd className="text-sky-600 truncate">{t(`resource_status__${resource.status}`)}</dd>
```
Add translations

File: src/components/Resource/ResourceList.tsx [hunk 5]
```diff
@@ -48,15 +48,125 @@ export default function ListView() {
...
+                  title={t("resource_status")}
+                  className={`w-fit mt-1 h-fit flex h-5 shrink-0 items-center rounded-full px-2 py-0.5 text-xs font-medium leading-4  ${
+                    resource.status === "APPROVED"
+                      ? "bg-sky-200"
+                      : "bg-yellow-200 text-yellow-500"
+                  }`}
+                >
+                  <CareIcon icon="l-truck" className="mr-2" />
+                  <dd className="text-sky-600">{resource.status}</dd>
+                </dt>
```
Comment (shivankacker) [inline src/components/Resource/ResourceList.tsx]: Can we use the `<Chip>` component here?

File: src/components/Resource/ResourceList.tsx [hunk 6]
```diff
@@ -48,15 +48,125 @@ export default function ListView() {
...
         </div>
       );
     }
 
-    return data.map((resource, i) => (
-      <div key={i} className="rounded-lg border border-secondary-300 bg-white">
-        <ResourceBlock resource={resource} />
+    return data.map((resource: ResourceModel, i) => (
+      <div key={i} className="w-full border-b-2 border-gray-100 col-span-6">
+        <div className="border-3  flex grid w-full gap-1 overflow-hidden bg-white p-4 shadow sm:grid-cols-1 md:grid-cols-1 lg:grid-cols-5">
```
Comment (shivankacker) [inline src/components/Resource/ResourceList.tsx]: We are moving away from shadows in our designs, lets use borders. `gray-300` or `gray-200` border would be fine

**Review by shivankacker — CHANGES_REQUESTED** [review]

File: src/components/Resource/ResourceList.tsx (hunk 5 above)
Comment (AnveshNalimela) [reply src/components/Resource/ResourceList.tsx]: Done

**Comment by khavinshankar** [comment]
@nihal467 can u please look into the cypress failure

**Review by shivankacker — APPROVED** [review]
LGTM, tests are failing

**Comment by AnveshNalimela** [comment]
I Dont Know Why they are failing. HCX claim specs are failing .

**Comment by AnveshNalimela** [comment]
@nihal467  Could you test and merge these PR..?

**Comment by AnveshNalimela** [comment]
@rithviknishad All the test are succesfull Could You trigger the merge

**Comment by nihal467** [comment]
LGTM

**Pull request merged**